meteor/scripts/i18n-compile-json.js

```diff
@@ -0,0 +1,51 @@
+const { promisify } = require('util')
+const glob = require('glob')
+const { spawn } = require('child_process')
+
+const pGlob = promisify(glob)
+
+/*************************************************
+
+This script goes through all of the languages (.po files)
+and compiles the json-files (used in production).
+
+**************************************************/
+
+;(async () => {
+	// List all po-files:
+	const poFiles = await pGlob('./i18n/*.po')
+
+	const languages = []
+	for (const poFile of poFiles) {
+		const mLanguage = poFile.match(/\/(\w+)\.po/)
+		if (mLanguage) languages.push(mLanguage[1])
+	}
+
+	console.log(`Found languages: ${languages.join(', ')}`)
+
+	for (const lng of languages) {
+		await runCmd(
+			`i18next-conv -l ${lng} -s i18n/${lng}.po -t public/locales/${lng}/translations.json --skipUntranslated`
+		)
+	}
+})().catch((e) => console.error(`ERROR: ${e}`))
```

Comment:
If this was an mjs file you wouldnt need this `;(async () => { ... })()` wrapper as top level await is possible.
You would have to change the `require()` to `import` too, but that is probably preferable as thats the syntax we use for typescript anyway